.gitignore

```diff
@@ -0,0 +1,167 @@
+list_of_recent_images.txt
+generated_build_manifest.json
+generated_build_versions_manifest.json
+generated_versions_table.md
```

Comment:
`self.review()` these 4 are the reasons for adding a git ignore file. The rest of this file is just the generic git ignore file from setting up a new github python project.

Reply:
alright I bit the bullet and added docker-images to .gitignore. Since the files had ballooned to 167 files to review.

Reply:
yeah we needed one.